src/index.hot.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { AppContainer as Hot } from 'react-hot-loader';
```

Comment:
IMO, renaming `` AppContainer `` to `` Hot `` is not better than using `` AppContainer ``, `` Hot `` doesn't convey that it's a HMRE related container.